content/en/platform/corda/5.0/developing-applications/getting-started/first-flow/_index.md

```diff
@@ -20,7 +20,7 @@ Many of the features will be familiar to those Developers who have written Corda
 ## MyFirstFlow Use Case
 
 The use case in the example flows is very simple:
-1. The initiating flow is called on the initiating node with another member of the application network, the recipient,  specified as the input argument.
+1. The initiating flow is called on the initiating node with another member of the {{< tooltip >}}application {{< /tooltip >}}, the recipient,  specified as the input argument.
```

Comment:
```suggestion
1. The initiating flow is called on the initiating node with another member of the {{< tooltip >}}application{{< /tooltip >}}, the recipient,  specified as the input argument.
```
That might have been the problem, I'm not 100%. Showing the underline no pop-up.